sw/device/lib/testing/keymgr_testutils.h

```diff
@@ -26,7 +26,7 @@ static const dif_keymgr_versioned_key_params_t kKeyVersionedParams = {
             0x322288d8,
             0xde919d54,
         },
-    .version = 0xaa,
+    .version = 0x1,
```

Comment:
@weicaiyang I had to make this change when I got some errors from keymgr saying that the KMAC input was invalid; the original version `0xaa` was indeed higher than the max version for the CreatorRootKey state (`0x11`). What worries me, though, is that this should _also_ have caused errors for the KMAC test, but apparently it didn't.

Reply:
I think it's because the kmac sideload enters OwnerInt before doing sideload operations. The max_key_version is set to 0xaa. So it's fine.

static const dif_keymgr_state_params_t kOwnerIntParams = {
    .binding_value = {0xe4987b39, 0x3f83d390, 0xc2f3bbaf, 0x3195dbfa,
                      0x23fb480c, 0xb012ae5e, 0xf1394d28, 0x1940ceeb},
    .max_key_version = 0xaa,
};

Reply:
The KMAC sideloading test runs the test twice, once in OwnerInt and once in CreatorRootKey:
https://github.com/lowRISC/opentitan/blob/d830b81d5aa9378116fe8a5336018416f6e29a84/sw/device/tests/sim_dv/keymgr_sideload_kmac_test.c#L91-L105

The first one, on line 97, should fail because of the key version, and does when I add random things such as extra state checks, or when I end the test after line 97. There's something weird going on there.

Reply:
You're right. Thanks for catching this.

I fixed it at #14690. The dif_keymgr accidentally clears the error status.